### DISCUSSION TITLE
[Feature] Mobile Upload Direct to Album / Archive

### DISCUSSION THREAD

**Discussion by werkkrew** [immich-app/immich#8357] [Feature Request]
### The feature

I would like the ability when selecting a phone folder location for upload / sync to Immich to be able to select a destination album as well as if it should be archived or not.

For example, lets say I have a folder on my phone where I regularly download silly memes from Reddit. I would like that to be able to get synced to an album called "Reddit Memes" and get automatically archived because while I want to preserve the images for posterity, I do not care to have them on my timeline.

The same logic could be applied to things like screenshots or other non-photograph style things.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678

**Reply by werkkrew** [reply]
I think I am asking for something a little bit different as 1) I would like to be able to select the destination album per source folder and not have it be "preserved" from the phone, and 2) the ability to archive. But yes, it does have much overlap.
